tests/test_minimal/test_tools/test_importing.py

```diff
@@ -0,0 +1,536 @@
+"""Tests for neuroconv.tools.importing module."""
+
+from neuroconv import get_format_summaries
+
+
+def test_guide_attributes():
+    """The GUIDE fetches this information from each class to render the selection of interfaces."""
+    summaries = get_format_summaries()
+
+    # Blocking assertion when adding new interfaces
+    for name, summary in summaries.items():
+        for key, value in summary.items():
+            assert value is not None, f"{name} is missing GUIDE related attribute {key}."
+            if isinstance(value, tuple):
+                assert len(value) > 0, f"{name} is missing entries in GUIDE related attribute {key}."
+
+    # For easier reference to global commonalities
+    assert summaries == {
```

Comment:
I think I'd rather check a few keys here rather than the entire dict. As it stands, we'd have to add each new data interface to this test

Reply:
@bendichter Would you be OK with at least having a static reference dumped to a JSON stored somewhere here (but not required for tests)?

It makes it a lot faster to reference at a glance whenever we have a question like 'why does this thing on the GUIDE look this way', instead of having to call code to generate it